src/main/cleanup_temp_dir.ts

```diff
@@ -25,7 +25,7 @@ export async function cleanupDraftTempDir() {
         `found old ${files.length} temporary draft files, trying to delete them now`
       )
       const promises = []
-      for (const file in files) {
+      for (const file of files) {
```

Comment:
without this I had on Linux errors: 
delete /tmp/deltachat-draft/0 - file not exists
delete /tmp/deltachat-draft/1 - file not exists
etc.